admin_guide/image_policy.adoc

```diff
@@ -108,12 +108,48 @@ The ImageStream "bar" is invalid:
 
 [[image-policy-configuring-the-image-policy-admission-plug-in]]
 == Configuring the ImagePolicy Admission Plug-in
+To configure which images can run on your cluster, configure the ImagePolicy
+Admission plug-in in the `*_master-config.yaml_*` file. You can set one or more
+rules as required. Following are examples of some of the rules you can set:
```

Comment:
I'd say "Following are examples of some of the rules you can set" is not needed #minimalism